mobile/apps/photos/lib/utils/gallery_util.dart

```diff
@@ -0,0 +1,18 @@
+import "package:photos/service_locator.dart";
+
+double get galleryCacheExtent {
```

Comment:
Let's keep it closer to the gallery namespace.. inside `ui/view/gallery`.

We can avoid putting such things inside utils.. Especially items that won't be utilized outside of the specific context (especially when there's single usage).